source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -46,6 +48,19 @@ In the following table, networking entities are listed with their implementation
 ### OVS networking model 
 ![OVS networking model](/feature/openvswitch/ovs_networking_model.svg)
 
+## Limitations
+
+Currently there are some limitations compared to Linux bridge [RFE OvS support](https://bugzilla.redhat.com/show_bug.cgi?id=1195208):
+
+- [Multiple gateways](https://bugzilla.redhat.com/show_bug.cgi?id=1383035) - Source routing is not implemented for the OVS switch type. This means that only management network can have gateway.
+- DHCPv6 - Dynimac address configuration for IPv6 is currently not supported. 
+- [Port mirroring](https://bugzilla.redhat.com/show_bug.cgi?id=1362492) -  Should be done via OVN.
+- [Host QoS](https://bugzilla.redhat.com/show_bug.cgi?id=1380271) - OvS supports different parameters to configure the limitations. Which is currently incompatible with solution used for Linux bridge.
```

Comment:
OvS supports different parameters to configure the limitations. Which is currently incompatible with solution used for Linux bridge.
->
 OvS supports different parameters for configuring QoS, Which are incompatible with the ones used for a Linux bridge.

Reply:
Done